### ISSUE TITLE
Cloudflare ddos protection looping

### ISSUE THREAD

**Issue by hushkamala** [cowlicks/privacypossum#269]
If you are reporting a bug, then providing the following is helpful:
* Waterfox Current 2020.09
* Possum version 2019.7.18

I saw the issue for cloudflare protected sites using captcha, i had to disable Privacy Possum for https://themusicfire.com/ and https://the-eye.eu/public/MSDN/ but these two sites don't use captchas, just some ddos cloudflare protection that you have to wait 5 seconds and the page refreshes itself and goes to the site.
But with Privacy Possum enabled the pages were refreshing in a loop never reaching the actual site.

      7722:
        received fingerprinting message from tab 'https://themusicfire.com/' for url 'Function'
      7721:
        received fingerprinting message from tab 'https://themusicfire.com/' for url 'Function'
      7720:
        received fingerprinting message from tab 'https://themusicfire.com/' for url 'Function'
      7719:
        received fingerprinting message from tab 'https://themusicfire.com/' for url 'Function'
      7718:
        cannot inject content script /js/initialize_contentscripts.js into url about:blank on tab 466 and frame 17179869820
      7717:
        cannot inject content script /js/contentscripts/fingercounting.js into url about:blank on tab 466 and frame 17179869820
      7716:
        cannot inject content script /js/bootstrap.js into url about:blank on tab 466 and frame 17179869820
      7715:
        cannot inject content script /js/initialize_contentscripts.js into url about:blank on tab 466 and frame 17179869818
      7714:
        cannot inject content script /js/contentscripts/fingercounting.js into url about:blank on tab 466 and frame 17179869818
      7713:
        cannot inject content script /js/bootstrap.js into url about:blank on tab 466 and frame 17179869818
      7712:
        received fingerprinting message from tab 'https://themusicfire.com/' for url 'Function'
      7711:
        Navigation error
      tabId: 466
      url: about:blank
      error: Error code 2152398850
      7710:
        Navigation error
      tabId: 466
      url: about:blank
      error: Error code 2152398850

Looks like Possum is removing the added url string: https://themusicfire.com/?__cf_chl_jschl_tk__=21d93aed01ce9d6407d6947838c9cf7766eb3b8c-1602526015-0-Aaa77uXrW-T-TN45j6yNB97YehFWqXnF3h07zrWXUln8dSEskK6zqb5TzNG_N91HABu5nT-Rh-HBUlNh36PjvfFDMHrbRMuhDlByLWGiP0AWm2xQoNV7qZcxPp2ObFiMLdLXSqu2fuZv_-qbWL-IhsBN-sMcwwCrdkHQZgC3D7E80zCplnzVeFW7VxOSQbx18B5mmwcbsOMiaCTKmZxS_CQqNjrRooyJx5g-wUsJptu18MkalROZU1AqYKc3thBX8Wk72Z_US81BQ6thPAkPJro

And maybe cloudflare needs it to verify someting, maybe related to cookies? Because i had to enable cookies for those sites. If i disable cookies theres a message to enable cookies and refresh page. After enabling cookies but also leaving Possum enabled it loops the same page. If i disble possum it goes to the actual site but with that huge url.

**Comment by RainmakerRaw** [comment]
Same here (Firefox 82, various Linux distros). I ended up just removing PP for now because so many of the sites I use were impacted.

**Comment by hushkamala** [comment]
Yeah i also switched to Firefox 82 but the problem remains which is expected. Although removing Possum is not what i did because it does its thing for everything else. Its a one time disable for every site, or disable and re-enable after the cloudflare thing and it works for some time even after page reloads and probably browser restarts.
